### DISCUSSION TITLE
🗼 五月底去了一趟日本，记录一下我的游记和攻略 | Mengke's blog - Mengke's coding journey

### DISCUSSION THREAD

**Discussion by giscus** [mk965/mengke.me#8] [Announcements]
# 🗼 五月底去了一趟日本，记录一下我的游记和攻略 | Mengke's blog - Mengke's coding journey

年初制定的目标中有一项是「带老婆出国旅行」，现在已经完成了。五月底我们去了一趟日本。我在这里记录一下我们的游记，也分享一些攻略 …

https://www.mengke.me/blog/202405/Travels_In_Japan

**Comment by ghost** [comment]
老哥好帅🤩🤩

**Reply by mk965** [reply]
你有一双善于发现美的眼睛🫣

**Comment by sherlockwhite** [comment]
一万6是两个人的花费吗

**Reply by mk965** [reply]
对，从离开家到回来的所有花费

**Comment by djabboc** [comment]
请问博主用的什么摄像设备？有经过后期调色吗？感觉颜色很好看

**Reply by mk965** [reply]
就iPhone 14 pro原相机

**Comment by mrmyn** [comment]
请问下博主是否需要换点日元现金带着过去呢

**Reply by mk965** [reply]
可以稍微带点，不用带太多，我们去的时候带了两万日元。

**Comment by mrmyn** [comment]
景点门票是通过什么平台订的

**Reply by mk965** [reply]
大部分景点都不要门票。迪士尼、环球是从携程还是去哪定的我忘了

**Comment by elrondwong** [comment]
拍得好看呢，正在做攻略，抄一抄

**Comment by jaywhen** [comment]
这个后续好棒🤩

**Comment by Jas0nG** [comment]
pretty cool!
